kube/services/revproxy/gen3.nginx.conf/cedar-wrapper-service.conf

```diff
@@ -0,0 +1,42 @@
+
+          location /cedar/ {
+
+            if ($csrf_check !~ ^ok-\S.+$) {
+                return 403 "failed csrf check";
+              }
+
+              set $authz_resource "/workspace";
```

Comment:
This still needs some revision but could use another set of eyes. I tried copying the authz logic from other endpoints but seems to be getting some different results. 

The goal is to protect authorize this endpoint to anyone with the `/cedar` resource (`/workspace` is a placeholder)

Reply:
what is the different results in here? This looks okay to me as long as we set the resource, method and service, and then pass to `/gen3-authz`